contracts/src/v0.8/dev/automation/2_1/KeeperRegistryLogicA2_1.sol

```diff
@@ -118,6 +118,33 @@ contract KeeperRegistryLogicA2_1 is
     return (upkeepNeeded, performData, upkeepFailureReason, gasUsed, fastGasWei, linkNative);
   }
 
+  /**
+   * @dev core node calls this function to call user contracts' mercury callback functions with proper gas limit
+   */
+  function mercuryCallback(
+    uint256 id,
+    bytes[] memory values,
+    bytes memory extraData
+  )
+    external
+    cannotExecute
+    returns (bool upkeepNeeded, bytes memory performData, UpkeepFailureReason upkeepFailureReason, uint256 gasUsed)
```

Comment:
can we ditch `gasUsed` ? I don't see it's usefulness. Or can someone explain to me why it's useful? cc @infiloop2